### ISSUE TITLE
Add svg picture 

### ISSUE THREAD

**Issue by kndidox** [python-openxml/python-docx#651]
Hi guys! Thanks for your great work! It seems that word has been providing support for svg for a couple of years now. I have tried adding an svg file using add_image but I get the following error message: 

> docx.image.exceptions.UnrecognizedImageError.

 I was wondering if you were planning to add support to svgs in the near future. I also tried adding the svg by manipulating the xmls, but I could not make it work either. It was a little too complicated for me. Thanks again!!

**Comment by scanny** [comment]
I don't believe Word supports SVG _natively_. What I believe it does is translate the SVG to EMF (Windows Meta File/Extended Meta File) format and embed that. Since EMF is a vector format, it looks just like the SVG.

The problem is that there is no SVG to EMF translator available in Python, at least the last time I looked. So this functionality (which would also be desirable for `python-pptx`) is not feasible at the moment.

**Comment by kndidox** [comment]
Hi @scanny thanks for your quick answer.

In working in MS Word 2016, as you surely know you can add a svg picture. I have unzipped the Word document to see how they manipulate this kind of files, then i realize that the xml drawing node is very similar for a svg/png. and in the media folder the svg is saved as it without changes, and all the references in the relationship file are to the svg.

So (and this is my supposition) it is technically possible give support to this without convertions.

If i can help you in anything let me know.

**Comment by BillChen2000** [comment]
I'm using Office 365 and I'm sure that Word supports svg format. Encountering the same issue, hope it can be fixed soon.

**Comment by ftomassetti** [comment]
Is this a duplicate of https://github.com/python-openxml/python-docx/issues/505 ?

**Comment by ftomassetti** [comment]
And of https://github.com/python-openxml/python-docx/issues/351

**Comment by mrgou** [comment]
It would be really awesome if this could be done. Word (and other Office applications) now support SVG _natively_ (they're now imported as editable vector images).

**Comment by juliuscrn** [comment]
Would love to see this feature as well.
